new-integration-tests/pom.xml

```diff
@@ -19,7 +19,7 @@
     <properties>
         <skipITs>false</skipITs>
         <wit.version>latest</wit.version>
-        <wdt.version>latest</wdt.version>
+        <wdt.version>1.8.1</wdt.version>
```

Comment:
why do you want to hard code the wdt.version?

Reply:
rolled it back to latest